### PR TITLE
Adding a scrub flag for sealdirectory function

### PR DESCRIPTION
**Pull request by norahuang**
Adding a scrub flag for sealdirectory function.

Fixes [AB#1468378](https://dev.azure.com/mseng/708e929f-6bd5-415a-8daf-25b1dac08dd8/_workitems/edit/1468378)

### CODE REVIEW

**Comment by narasamdya** [comment]
Please fix the typo in the title of PR before using it as a commit title.

File: Public/Src/FrontEnd/UnitTests/Libs/Prelude.AmbientHacks.ts [hunk 1]
```diff
@@ -59,7 +59,7 @@ namespace _PreludeAmbientHack_Transformer {
     export declare function writeData(destinationFile: Path, content: any, tags?: string[], description?: string): DerivedFile;
     export declare function writeAllLines(destinationFile: Path, contents: any[], tags?: string[], description?: string): DerivedFile;
     export declare function writeAllText(destinationFile: Path, content: string, tags?: string[], description?: string): DerivedFile;
-        export declare function sealDirectory(root: Directory, files: File[], tags?: string[], description?: string): FullStaticContentDirectory;
+    export declare function sealDirectory(root: Directory, files: File[], tags?: string[], description?: string, scrub?: boolean): FullStaticContentDirectory;
```
Comment (dannyvv) [inline Public/Src/FrontEnd/UnitTests/Libs/Prelude.AmbientHacks.ts]: >root [](start = 42, length = 4)

After chatting, we should probably clean this api up and make it take an object literal.
I'll file a workitem on myself to do this in the future.
Let me know if you are going to add more.

Comment (norahuang) [reply Public/Src/FrontEnd/UnitTests/Libs/Prelude.AmbientHacks.ts]: I don't see it will happen in the near future.

**Pull request merged**